packages/client/package.json

```diff
@@ -28,7 +28,7 @@
     "@types/jwt-decode": "^2.1.0",
     "@types/mousetrap": "^1.6.2",
     "@types/ms": "^0.7.30",
-    "@types/react": "16.9.11",
+    "@types/react": "^17.0.43",
```

Comment:
aha, just curious about `^18.0` for `@types/react`?

Reply:
good catch, when I initially created the PR that was the latest version, will update 👍